`docker build` is failed

https://circleci.com/gh/sue445/dockerfile-heroku-cli/223

```
$ docker run -it --rm node:8-alpine sh

/ # npm install -g heroku
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @heroku-cli/plugin-certs-v5@undefined
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-06-20T07_55_13_040Z-debug.log

/ # cat /root/.npm/_logs/2018-06-20T07_55_13_040Z-debug.log

(snip)

42 http fetch GET 304 https://registry.npmjs.org/@heroku-cli%2fplugin-certs-v5 203ms (from cache)
43 silly fetchPackageMetaData error for @heroku-cli/plugin-certs-v5@latest No matching version found for @heroku-cli/plugin-certs-v5@undefined
44 verbose type tag
45 verbose stack @heroku-cli/plugin-certs-v5: No matching version found for @heroku-cli/plugin-certs-v5@undefined
45 verbose stack     at pickManifest (/usr/local/lib/node_modules/npm/node_modules/pacote/node_modules/npm-pick-manifest/index.js:65:11)
45 verbose stack     at fetchPackument.then.packument (/usr/local/lib/node_modules/npm/node_modules/pacote/lib/fetchers/registry/manifest.js:52:18)
45 verbose stack     at tryCatcher (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
45 verbose stack     at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:512:31)
45 verbose stack     at Promise._settlePromise (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:569:18)
45 verbose stack     at Promise._settlePromise0 (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:614:10)
45 verbose stack     at Promise._settlePromises (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:693:18)
45 verbose stack     at Async._drainQueue (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:133:16)
45 verbose stack     at Async._drainQueues (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:143:10)
45 verbose stack     at Immediate.Async.drainQueues (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:17:14)
45 verbose stack     at runCallback (timers.js:810:20)
45 verbose stack     at tryOnImmediate (timers.js:768:5)
45 verbose stack     at processImmediate [as _immediateCallback] (timers.js:745:5)
46 verbose cwd /
47 verbose Linux 3.10.0-693.11.1.el7.x86_64
48 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "heroku"
49 verbose node v8.11.3
50 verbose npm  v5.6.0
51 error code ETARGET
52 error notarget No matching version found for @heroku-cli/plugin-certs-v5@undefined
53 error notarget In most cases you or one of your dependencies are requesting
53 error notarget a package version that doesn't exist.
54 verbose exit [ 1, true ]
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,11 +1,10 @@
-FROM node:8-alpine
+FROM debian:stable-slim
 
 MAINTAINER sue445 <sue445@sue445.net>
 
-RUN apk --update --no-cache add bash git openssh-client
+RUN apt-get update \
+ && apt-get install -y curl xz-utils git \
+ && apt-get clean \
+ && apt-get autoclean
 
-RUN npm install -g heroku \
- && rm -rf /tmp/* /root/.npm \
- && cd /usr/local/lib/node_modules/npm/ \
- && rm -rf man doc html *.md *.bat *.yml changelogs scripts test AUTHORS LICENSE Makefile \
- && find /usr/local/lib/node_modules/ -name test -o -name tests -o -name .bin -type d | xargs rm -rf
+RUN curl https://cli-assets.heroku.com/install.sh | sh

